virtual_ecosystem/models/hydrology/hydrology_model.py

```diff
@@ -387,8 +379,8 @@ def update(self, time_index: int, **kwargs: Any) -> None:
             time_index=time_index,
             days=days,
             seed=seed,
-            layer_roles=self.layer_structure.layer_roles,
-            soil_layer_thickness=self.soil_layer_thickness,
+            layer_structure=self.layer_structure,
+            soil_layer_thickness=self.soil_layer_thickness_mm,
```

Comment:
@vgro - the `LayerStructure` provides soil layer thickness in metres, so I've renamed and converted here. It probably makes sense for `LayerStructure.soil_layer_thickness` to switch over to mm. @jacobcook1995 doesn't have strong feelings.

Reply:
I think it is fine in meters and convert here, I use it in the soil energy balance in meters, so one of them must be converted and to be consistent with the layer heights, meters works better in the LayerStructure I think